src/backends/swift/objects.ts

```diff
@@ -241,12 +241,16 @@ export async function listObjects(
   const delimiter = query.delimiter ? query.delimiter[0] : undefined;
   const prefix = query.prefix ? query.prefix[0] : undefined;
   const maxKeys = query["max-keys"] ? Number(query["max-keys"][0]) : undefined;
+  const continuationToken = query["continuation-token"]
+    ? query["continuation-token"][0]
+    : undefined;
```

Comment:
undefined has a specific meaning in typescript, can we use the default "null" value instead?